focus-ios/focus-ios-tests/XCUITest/PageActionMenuTest.swift

```diff
@@ -31,6 +33,7 @@ class FindInPageTest: BaseTestCase {
         waitForNoExistence(app.buttons["FindInPage.close"])
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/395022
```

Comment:
I just realized this test is Smoke test in test rail and runs in bitrise as part of the smoketest suite. Please add the // Smoketest